static/js/subs.js

```diff
@@ -3,10 +3,16 @@ var subs = (function () {
 var exports = {};
 
 function add_admin_options(sub) {
+    if (sub.subscribed == true) {
+        var can_moderate = _.contains(sub.permissions, 'can_moderate');
```

Comment:
Nit: you don't really need the `can_moderate` variable. I'd probably just put the expression on the right inline in the assignment of `can_moderate_stream` below